background_scripts/completion_search.coffee

```diff
@@ -42,7 +42,7 @@ CompletionSearch =
 
   # The amount of time to wait for new requests before launching the current request (for example, if the user
   # is still typing).
-  delay: 100
+  delay: 0
```

Comment:
> but a 100ms delay does nothing to reduce the number of requests

Yes it does.  Run with the debug turned on and you'll see them being cancelled.

There are two types of rate limiting going on.  They're different.  On the front end we throttle updates, but allow them to proceed once every 150ms.  On the back end we cancel completion requests completely if the user is still typing (to be good net citizens).